clearml_agent/commands/worker.py

```diff
@@ -2876,19 +2876,22 @@ def freeze_task_environment(self, task_id=None, requirements_manager=None,
             self.log_traceback(e)
         return freeze
 
-    def _install_poetry_requirements(self, repo_info):
-        # type: (Optional[RepoInfo]) -> Optional[PoetryAPI]
+    def _install_poetry_requirements(self, repo_info, lockfile_path):
+        # type: (Optional[RepoInfo], Path) -> Optional[PoetryAPI]
         if not repo_info:
             return None
         try:
             if not self.poetry.enabled:
                 return None
-            self.poetry.initialize(cwd=repo_info.root)
-            api = self.poetry.get_api(repo_info.root)
+
+            self.poetry.initialize(cwd=lockfile_path)
+            api = self.poetry.get_api(lockfile_path)
             if api.enabled:
                 print('Poetry Enabled: Ignoring requested python packages, using repository poetry lock file!')
                 api.install()
                 return api
+            
+            print(f"Could not find pyproject.toml or poetry.lock file in {str(lockfile_path)} \n")
```

Comment:
No need for `str()` call (f-string automatically converts to string), also I think `\n` can be removed.

Reply:
Removed the `str()` call, gets converted automatically indeed, thought it would then show `PosixPath(... ) ` but it doesn't indeed.

> also I think \n can be removed.

I do think the \n can be nice in the logs, because if this shows up then right after the pip import starts and it can be hard to spot this line otherwise I feel:

```console
type: git
url: redacted
branch: HEAD
commit: redacted
root: redacted


Configuration file exists at redacted, reusing this directory.

Consider moving configuration to redacted, as support for the legacy directory will be removed in an upcoming release.
Could not find pyproject.toml or poetry.lock file in /my/root/directory/repo.git 

Ignoring pip: markers 'python_version >= "3.10"' don't match your environment
Collecting pip<20.2
  Using cached pip-20.1.1-py2.py3-none-any.whl (1.5 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 23.0.1
```

wdyt?

Reply:
Sure, no problem